app/modules/intelligence/agents/chat_agents/pydantic_agent.py

```
@@ -251,20 +256,29 @@ def _create_task_description(
         image_context = ""
-        if ctx.has_images():
+        if ctx.has_images() or ctx.has_documents():
             all_images = ctx.get_all_images()
             image_details = []
             for attachment_id, image_data in all_images.items():
                 file_name = image_data.get("file_name", "unknown")
                 file_size = image_data.get("file_size", 0)
                 image_details.append(f"- {file_name} ({file_size} bytes)")
+            document_details = []
+            all_documents = ctx.get_all_documents()
+            for attachment_id, doc_data in all_documents.items():
```

Ruff (0.15.9)

[warning] Loop control variable `attachment_id` not used within loop body

Rename unused `attachment_id` to `_attachment_id`

(B007)